packages/atomic/src/components/insight/atomic-insight-tabs/atomic-insight-tabs.pcss

```diff
@@ -0,0 +1,7 @@
+@import '../../../global/global.pcss';
+
+:host {
+  grid-column: 1 / 5;
```

Comment:
`grid-column` is only relevant because the parent component is a grid in `insights.html`. Elsewhere, we don't necessarily know that this may be the case.

I would move `grid-column` to `insights.html`.

Reply:
yeah you're right. this should be conditional on the layout system that will come later...